doc/source/management/data_at_rest_encryption.rst

```diff
@@ -79,15 +89,36 @@ To change the default value use the following syntax:
 
 Here, **NEW_ID** is an unsigned 32-bit integer.
 
+
 .. _data-at-rest-encryption.innodb-system-tablespace:
 
+Encryption threads
+========================================================================
+
+Adding explicit encryption in an `ALTER TABLE` or `CREATE TABLE` statement with the `ENCRYPTION`='N' clause excludes the table or tablespace from encryption threads. The INFORMATION_SCHEMA table: INNODB_TABLESPACE_ENCRYPTION displays the information for an excluded table or excluded tablespace. This table is currently used to show information about the tables encrypted by encryption threads. Due to changes in 8.0.16, the `SHOW CREATE TABLE` statement may not verify if the `ENCRYPTION`=’N’ phrase was explicitly specified. 
+
+ MariaDB only allows you to exclude innodb_file_per_table tablespace. In Percona, all tablespaces with `ENCRYPTION`='N' should be excluded from encryption threads. This approach solves the issue of upgrading from 5.7 to 8.0.16. One issue was the explicit encryption would be based on the `CREATE INFO` flags from .frm file or crypt_data must be read before the upgrade. This requirement is not longer with with`INNODB_TABLESPACE_ENCRYPTION`.
+
+ The `TABLE_ENCRYPTION_ADMIN` privilege allows the user to `CREATE TABLE ENCYPTION`='N' in `DATABASE` with `DEFAULT ENCRYPTION`='Y'. The `default-table-encryption` is used by `DATABASE` and `TABLESPACES`. Whe you create a new DATABASE and do not specify the DEFAULT ENCRYPTION, the DEFAULT ENCRYPTION is set to the same value as the default-table-encryption. 
+
+Before 8.0.16, only an explicit `ENCRYPTION`='N' was visible in SHOW CREATE TABLE. With 8.0.16, a table or a tablespace is not allowed to have an empty ENCRYPTION clause and all unencrypted tables or tablespaces have the setting `ENCRYPTION`='N'. The `SHOW CREATE TABLE ENCRYPTION`=’N’ is only displayed when the encryption of the table or the tablespace is different from schema’s `DEFAULT ENCRYPTION` setting. This is slightly different than MySQL, when the `ENCRYPTION`='N' is only returned from the DATABASE DEFAULT ENCRYPTION. 
```

Comment:
This is not correct. Both MySQL and PS show ENCRYPTION='N' in SHOW CREATE TABLE when it is different from schema's DEFAULT ENCRYPTION. The current behavior is described by PS-5821.

Reply:
will fix. Thanks.